### PR TITLE
Fixed a couple of test scripts.

### PR DESCRIPTION
**Pull request by vpatryshev**
First, reporting better the failures I observed.
Second, make several attemps of H2O resuscitation before hanging up.

### CODE REVIEW

File: scripts/run.py [hunk 1]
```diff
@@ -1409,14 +1413,17 @@ def check_clouds(self):
...
         """
-        time.sleep(3)
-        print("Checking cloud health...")
-        for c in self.clouds:
-            if self._h2o_exists_and_healthy(c.get_ip(), c.get_port()):
-                print("Node {} healthy.".format(c))
-            else:
-                print("Node with IP {} and port {} NOT HEALTHY" .format(c.get_ip(),c.get_port()))
-                # should an exception be thrown?
+        for i in range(5):
```
Comment (michalkurka) [inline scripts/run.py]: What is the reason for repeating the checks?

The cloud is only checked after the tests and checking has not other effect than printing a message to a log file.

File: h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py [hunk 2]
```diff
@@ -72,7 +72,10 @@ def random_grid_model_seeds_PUBDEV_4090():
     model_len = min(len(air_grid1), len(air_grid2))
     correct_model_seeds = list(range(search_criteria_tune1["seed"], search_criteria_tune1["seed"]+model_len))
     assert model_seeds1[0:model_len]==model_seeds2[0:model_len], "Random gridsearch seed is not set correctly as model seeds..."
-    assert model_seeds1==correct_model_seeds, "Random gridsearch seed is not "
+    # this assert was failing; have to add more diagnostics
+    expected1 = ','.join(correct_model_seeds)
+    actual1 = ','.join(mdel_seeds1)
+    assert model_seeds1==correct_model_seeds, "Model seeds are not correct: expected %s; got %s" % (expected1, actual1)
```
Comment (wendycwong) [inline h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py]: Vlad:  I will incorporate your change on my PR.  If you can remove this from yours, that will be great.

File: scripts/run.py (hunk 1 above)
Comment (wendycwong) [inline scripts/run.py]: Why do you want to perform the health checks 5 times instead of just once?  Will make the tests run longer.

Comment (vpatryshev) [inline scripts/run.py]: Right. Then my question will be: WHY check the cloud? Makes no sense, if the tests have already passed. The effect actually is that the test is reported as failing. Let's not check anything at all at this moment. Either the tests failed, or they passed, that's it.

Comment (wendycwong) [inline scripts/run.py]: Ask Tomas, he may know.

**Pull request closed without merging**